content/reference/agent-authentication.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Agent authentication types'
+categories: ["reference"]
+description: Authentication types for the Rhize agent
+weight: 900
+menu:
+  main:
+    parent: reference
+    identifier:
+---
+
+The Rhize agent collects data emitted that is emitted in the manufacturing process and makes this data visible in the Rhize system.
+
+It works by connecting to equipment or groups of equipment that run over protocols such as OPC UA,
+and then reading and writing tags to re-publish them to NATS.
```

Comment:
```suggestion
and then subscribing to tags to re-publish them to NATS.
```

Then I'd add something about also providing datasource reads/writes and method calls via BPMN.

Reply:
[185a9c8](https://github.com/libremfg/libremfg.github.io/pull/26/commits/185a9c8ddb1d7b7118b7761608a5d2da6bc6cab5) Expands the intro a bit